lib/Core/Executor.cpp

```diff
@@ -4091,6 +4101,19 @@ int *Executor::getErrnoLocation(const ExecutionState &state) const {
 }
 
 
+void Executor::dumpPTreeCSV(PTreeNode *n) {
+  assert(n && n->left && n->right);
+  if (!::DumpProcessTree) return;
+
+  char name[32];
+  snprintf(name, sizeof(name),"ptree%08d.csv", (int) stats::instructions);
+  static auto os = interpreterHandler->openOutputFile(name);
```

Comment:
Martin doesn't like static variables. Probably better to make it a member of executor and give it a proper name.

Also doesn't this open a new file whenever dumPTreeCSV is called? Looks wrong to me